src/js/components/Data/stories/SpaceX.js

```diff
@@ -0,0 +1,223 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Search isn't working on the Data/SpaceX or Data/InfiniteScroll stories


https://user-images.githubusercontent.com/54560994/207738414-33e94460-b155-44f9-8fbd-2cc424f95828.mp4